src/plugins/assetUploader/README.md

```diff
@@ -0,0 +1,157 @@
+# NFT Asset Uploader Guide
+
+## Pre-requisite
+
+Before you begin, make sure you have the following:
+
+1. SDK version:
+   Ensure you have version 1.9.1-tmp.0 of `@aptos-labs/ts-sdk`.
+   [SDK Package](https://www.npmjs.com/package/@aptos-labs/ts-sdk/v/1.9.1-tmp.0?activeTab=versions)
+
+2. Install the SDK with `pnpm`:
+   ```bash
+   pnpm install @aptos-labs/ts-sdk@1.9.1-tmp.0
+   ```
+
+3. Install the Irys SDK with `npm`:
+   ```bash
+   npm install @irys/sdk
+   ```
+
+4. Prepare your NFT media and metadata files:
+   Begin with the metadata template and replace the content as necessary.
+   ```json
+   {
+       "name": "Asset Name",
+       "description": "Lorem ipsum...",
+       "image": "",
+       "properties": {
+           "simple_property": "example value"
+       }
+   }
+   ```
+   Supported media types include: `png`, `jpg`, `jpeg`, `gif`, etc.
+
+   Ensure that the file name in the metadata corresponds to the media file name.
+
+   Each metadata file and corresponding NFT media should be unique.
+
+
+## How to Use
+
+Below is a sample TypeScript script that demonstrates how to use the uploader:
+
+```ts
+/* eslint-disable no-console */
+import { Account, AptosConfig, Network, Aptos, AssetUploader } from "@aptos-labs/ts-sdk";
+import * as fs from "fs";
+import * as path from "path";
+
+const example = async () => {
```

Comment:
feels a bit redundant to put the same example here, I would keep the example file only and link to it from here